libstuff/libstuff.h

```diff
@@ -712,6 +712,14 @@ inline string SUNQUOTED_TIMESTAMP(uint64_t when) { return SComposeTime("%Y-%m-%d
 inline string STIMESTAMP(uint64_t when) { return SQ(SUNQUOTED_TIMESTAMP(when)); }
 inline string SUNQUOTED_CURRENT_TIMESTAMP() { return SUNQUOTED_TIMESTAMP(STimeNow()); }
 inline string SCURRENT_TIMESTAMP() { return STIMESTAMP(STimeNow()); }
+inline string SCURRENT_TIMESTAMP_MS() {
```

Comment:
I should have done this in the previous review, but I'm also gonna ask you to make this not `inline` and move the implementation into libstuff.cpp

Reply:
Would `STime.cpp` be a more appropriate place to put this?

Reply:
It could be. I don't have strong feelings on it.